.github/ISSUE_TEMPLATE/bad_codegen.yml

```diff
@@ -0,0 +1,38 @@
+name: Report bad codegen
+description: Report cases where triton is generating sub-optimal (but functionally correct) PTX/LLVM IR
```

Comment:
nit: "bad codegen" may be a bit ambiguous name, maybe we can mention performance in the title